pkgs/applications/audio/eq10q/default.nix

```diff
@@ -9,7 +9,9 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ cmake fftw gtkmm2 libxcb lv2 xorg.libpthreadstubs xorg.libXdmcp xorg.libxshmfence ];
+  # lv2 1.18.0 has a breaking change, and the usual patch doesn't work here, see:
+  # https://sourceforge.net/p/eq10q/bugs/23/
```

Comment:
Looks like it uses `const _LV2UI_Descriptor` instead of `const struct _LV2UI_Descriptor`:

https://sourceforge.net/p/eq10q/code/HEAD/tree/trunk/gui/eq10q_ui.cpp#l41

Reply:
Thanks, done!